config/src/main/java/com/typesafe/config/impl/ConfigNodeComplexValue.java

```diff
@@ -24,4 +24,29 @@
         }
         return tokens;
     }
+
+    protected ConfigNodeComplexValue indentText(AbstractConfigNode indentation) {
+        ArrayList<AbstractConfigNode> childrenCopy = new ArrayList<AbstractConfigNode>(children);
+        for (int i = 0; i < childrenCopy.size(); i++) {
+            if (childrenCopy.get(i) instanceof ConfigNodeSingleToken &&
```

Comment:
if you do `AbstractConfigNode child = childrenCopy.get(i)` at the top here, you'd avoid a lot of noisy `childrenCopy.get(i)` repetition